pkgs/development/libraries/mimalloc/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib, stdenv, fetchFromGitHub, cmake, ninja
 , fetchpatch
 , secureBuild ? false
+, overrideMalloc ? null
```

Comment:
```suggestion
, overrideMalloc ? false
```

Reply:
I don't think I agree with this change. Note that the upstream default is `overrideMalloc = true / ON`, so changing this to false by default will almost certainly break (or end up falling back to glibc's allocator) whatever is currently depending on mimalloc as it is. This would apply to `rust-analyzer` with `useMimalloc = true`, any packages or development environments that use mimalloc but which are not in nixpkgs, and the system-wide `environment.memoryAllocator.provider` option.

If you prefer this style then I'd be OK with setting it to `true`, rather than `false`, instead of the `null` approach which falls back to whatever the upstream mimalloc does at any point in time.

Reply:
I really messed up with this review, sorry. Yes I meant to put `true` actually here.